src/remote-config/remote-config-api-client.ts

```diff
@@ -140,16 +134,11 @@ export class RemoteConfigApiClient {
     this.validateRemoteConfigTemplate(template);
     return this.sendPutRequest(template, template.etag, true)
       .then((resp) => {
+        // validating a template returns an etag with the suffix -0 means that your update 
```

Comment:
Do we ever check for this `-0` suffix?

Reply:
We don't necessarily check for `-0` suffix here. If the response is not an error (200 OK) and contains a valid etag, we can safely assume that the template was successfully validated. All other cases should return an error code. I will add an additional check for the `-0` suffix in a separate PR for good measure :)